packages/EasyCore/src/Helpers/CleanerInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Helpers;
+
+interface CleanerInterface
```

Comment:
Let's name it as `StringsTrimmerInterface`

Reply:
Done.